examples/basics/custom-type/README.md

```diff
@@ -8,5 +8,5 @@ This example will show you how to create and interact with custom object types.
 
 ## Documentation
 
-* [Generic objects](https://help.qlik.com/en-US/sense-developer/June2017/Subsystems/EngineAPI/Content/GenericObject/overview-generic-object.htm)
-* [Properties of generic objects](https://help.qlik.com/en-US/sense-developer/June2017/Subsystems/EngineAPI/Content/GenericObject/Properties.htm)
+* [Generic object model](https://qlik.dev/libraries-and-tools/enigmajs/#generic-object-model)
```

Comment:
I'm a bit hesitant to link to a `enigma.js`, would be a bit confusing. I think we should keep the help.qlik.com link for now and instead work on improving the `enigma-go` documentaion in the dev-portal instead.

Reply:
you're right (I thought it's under qix), I reverted the change 

this page (the Generic Object model) should have been more generic and been under [QIX introduction](https://qlik.dev/apis/json-rpc/qix)